scripts/test_pypi_packages.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3.8
```

Comment:
chmod 0755

Reply:
Should I change something here?

Reply:
In the UNIX shell you should execute `chmod +x scripts/test_pypi_packages.py` and then commit the change (git detects file mode changes).

Reply:
Done!